cmd/minikube/cmd/start.go

```diff
@@ -1131,6 +1131,12 @@ func getKubernetesVersion(old *cfg.MachineConfig) (string, bool) {
 		} else { // .. otherwise use the default version
 			paramVersion = constants.DefaultKubernetesVersion
 		}
+	} else if paramVersion == "stable" || paramVersion == "latest" {
+			if paramVersion == "stable" {
+				paramVersion = constants.DefaultKubernetesVersion
+			} else {
+				paramVersion = constants.NewestKubernetesVersion
+			}
```

Comment:
These can just be two separate `else if` statements instead of nested `if`s.

Reply:
Ok.Thanks for the feed back, changed it